config/params/config.go

```diff
@@ -280,6 +280,11 @@ type BeaconChainConfig struct {
 	AttestationSubnetPrefixBits     uint64          `yaml:"ATTESTATION_SUBNET_PREFIX_BITS" spec:"true"`     // AttestationSubnetPrefixBits is defined as (ceillog2(ATTESTATION_SUBNET_COUNT) + ATTESTATION_SUBNET_EXTRA_BITS).
 	SubnetsPerNode                  uint64          `yaml:"SUBNETS_PER_NODE" spec:"true"`                   // SubnetsPerNode is the number of long-lived subnets a beacon node should be subscribed to.
 	NodeIdBits                      uint64          `yaml:"NODE_ID_BITS" spec:"true"`                       // NodeIdBits defines the bit length of a node id.
+
+	// Blobs Values
+	MaxBlobsPerBlock           int `yaml:"MAX_BLOBS_PER_BLOCK" spec:"true"`            // MaxBlobsPerBlock defines the max blobs could exist in a block.
```

Comment:
I think having this as a public config value is going to be a potential footgun or lead to misuse. Is there a way that this can _only_ be accessed via the helper method? If there is a single place in the production codepath that is using `params.BeaconConfig().MaxBlobsPerBlock` instead of `params.BeaconConfig().MaxBlobCount(slot)` then it's going to be a nasty bug at the fork.